docs/UserGuide.md

```diff
@@ -57,12 +57,31 @@ This guide provides you with in-depth information on setting up AniChan and how
 
 <br/>
 
+### 1.2 Using this guide
+
+Along the way you might encounter several icons. These icons can help provide several types of information that may be useful to you. 
+
+> :bulb: Take note when you see this icon, as it might tell you something important.
+
+> :memo: This icon represents a friendly tip that might be useful when using our application.
+
+> :warning: A caution that you have to keep in mind when using the command
+
+Lastly, text that are blue like this [example](#12-using-this-guide), are clickable links that will bring you to the relevant part of this user guide. 
+
+<br/>
+
 ## 2. Quick Start
 1.  Ensure that you have Java 11 or above installed.
 2.  Download the latest version of **AniChan** from [here](https://github.com/AY2021S1-CS2113T-F12-2/tp/releases/tag/V2.0).
 3.  Copy the file to the folder you want to use as the home folder for **AniChan**.
 4.  Open **command prompt** or **terminal** and change directory into the folder. Run `java -jar AniChan.jar`.
-5.  Type a command into the command prompt and press `Enter` to execute it. For example, typing `help` and pressing `Enter` will display the help message.
+5.  You will be greeted by a prompt for your name and gender as shown below.
+6.  Upon completing these 2 prompts, you can begin using **AniChan**!
```

Comment:
Could it be confusing to mention 2 prompts when line 79 states `a prompt`?

Reply:
Hmm, you are right, I will amend this in the next PR. Thank you!